### ISSUE TITLE
fix security vulnerability in axios and update dependencies

### ISSUE THREAD

**Issue by LeichtF** [siddharthkp/bundlesize#387]
**Do you want to request a *feature* or report a *bug*?** 
bug

**What is the current behavior?**
* axios < 1.6.8 has various security vulnerabilities
* most of the other dependencies are outdated

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information.**
- node version 10.24.1 
- npm version npm@6.14.12
- Operating system
- bundlesize version 0.18.2
- CI you are using

**Comment by LeichtF** [comment]
@falsyvalues  I disagree. #382 upgraded axios from 0.21.1 to 1.6.2 and github-build from 1.2.2 to 1.2.4.
#388 is taking care of more packages and also upgrades to newer versions. axios should be upgraded to 1.6.4 or above to include the latest security fixes (see release notes for axios [1.6.4](https://github.com/axios/axios/releases/tag/v1.6.4))

**Comment by falsyvalues** [comment]
@LeichtF Oh, let me correct that. Anyway its up to maintainer if he wants to be on the edge with other dependencies.
